nucliadb/src/nucliadb/ingest/orm/brain.py

```diff
@@ -427,62 +428,62 @@ def _set_resource_relations(self, basic: Basic, origin: Optional[Origin]):
     def _set_resource_labels(self, basic: Basic, origin: Optional[Origin]):
         if origin is not None:
             if origin.source_id:
-                self.labels["o"] = [origin.source_id]
+                self.labels["o"].add(origin.source_id)
```

Comment:
This one changes behaviour if `self.labels["o"]` was prepopulated. Is this ok?

Reply:
changed as it was before 👍🏼